.github/workflows/build-packages.yml

```
@@ -193,17 +184,17 @@ jobs:
+      uses: sigstore/cosign-installer@cad07c2e89fa2edd6e2d7bab4c1aa38e53f76003 # v4.1.1
 
     - name: Sign package
       id: sign_package
-      run : |
-        ls ${{github.workspace}}/${{ matrix.asset_path }}
-        sudo mv ${{github.workspace}}/${{ matrix.asset_path }}/flex* ${{github.workspace}}/${{ matrix.asset_path }}/flex${{ matrix.asset_prefix }}${{ steps.previoustag.outputs.version }}${{ matrix.asset_suffix }} || echo "no need to rename"
-        echo '%_gpg_name moauto (automation) <54212639+mo-auto@users.noreply.github.com>' >> ~/.rpmmacros
-        ${{ matrix.sign_cmd }} ${{github.workspace}}/${{ matrix.asset_path }}/flex${{ matrix.asset_prefix }}${{ steps.previoustag.outputs.version }}${{ matrix.asset_suffix }}
-        gpg --armor --detach-sign ${{github.workspace}}/${{ matrix.asset_path }}/flex${{ matrix.asset_prefix }}${{ steps.previoustag.outputs.version }}${{ matrix.asset_suffix }}
-        ls ${{github.workspace}}/${{ matrix.asset_path }}
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:3:1: SUFFIX appears unused. Verify use (or export if used externally)

(shellcheck)